src/averageAges.js

```diff
@@ -14,12 +14,26 @@
  *
  * @return {number}
  */
+
 function calculateMenAverageAge(people, century) {
   // write code here
   // learn how to use array methods like .filter .map .some .every .find .reduce
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  const manArr = people.filter(person => person.sex === 'm');
+
+  let yearsArr;
+
+  if (century) {
+    yearsArr = manArr
+      .filter(person => Math.ceil(person.died / 100) === century)
+      .map(person => +person.died - +person.born);
+  } else {
+    yearsArr = manArr.map(person => +person.died - +person.born);
+  };
```

Comment:
move map(person => +person.died - +person.born) into a function to not repeat yourself.
Use destructure assignment